DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -12,7 +12,7 @@
    <version ClassVersion="10" checksum="877294307"/>
   </class>
   <class name="SiTrackerGSMatchedRecHit2D" ClassVersion="14">
-    <version ClassVersion="14" checksum="3882739671"/>
+    <version ClassVersion="14" checksum="3518158739"/>
```

Comment:
@lveldere please make sure 76X classversion-checksum pairs match the 75X.
It looks like we have diverged in version 14.
So, this change should be
```diff
 -    <version ClassVersion="14" checksum="3882739671"/> 
+    <version ClassVersion="14" checksum="1823455267"/>
 +    <version ClassVersion="15" checksum="3518158739"/> 
```
this will loose support for the 76X few days old ```<version ClassVersion="14" checksum="3882739671"/>```